tensorflow/python/kernel_tests/sparse_tensor_dense_matmul_op_test.py

```diff
@@ -132,8 +132,10 @@ def testShapeInference(self):
       sparse_ops.sparse_tensor_dense_matmul(x_st_shape_inconsistent, y)
 
   def testInvalidIndicesForSparseTensorDenseMatmul(self):
-    # Note: use_gpu=False because nice errors are only returned from CPU kernel.
-    with self.test_session(use_gpu=False):
```

Comment:
I think the intention here was to run testInvalidIndicesForSparseTensorDenseMatmul on CPU only